content/workshops/rhel_8/_index.adoc

```diff
@@ -0,0 +1,75 @@
+---
+title: Red Hat Enterprise Linux 8
+menu:
+  main:
+    parent: workshops
+    pre: fa fa-magic
+---
+
+:domain_name: redhatgov.io
+:workshop_prefix: workshop
+:tower_url: https://{workshop_prefix}.*student_number*.{domain_name}
+:ssh_url: https://{workshop_prefix}.*student_number*.{domain_name}:9090
+
+:icons: font
+:iconsdir: http://people.redhat.com/~jduncan/images/icons
+:imagesdir: /workshops/rhel_8/images
+
+= Red Hat Enterprise Linux 8 Workshop
+
+== Workshop Description
+
+This workshop will cover the features, specifications and improvements that comprise the release of version 8 of Red Hat Enterprise Linux (RHEL).  RHEL 8 brings many improvements over previous releases, including:
+
+* Application streams, which allow for the use of bleeding edge interpreters, compilers and libraries, while maintaining full backwards compatibility and stability of the system
```

Comment:
Line 24 Replace “allow for” with “enables”

Reply:
Fixed